Xamarin.Essentials/Screenshot/Screenshot.android.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Android;
+using Android.App;
+using Android.Content;
+using Android.Content.Res;
+using Android.Graphics;
+using Android.OS;
+using Android.Telephony;
+using Android.Util;
+using Android.Views;
+using Java.Util;
+using Environment = Android.OS.Environment;
+using Path = System.IO.Path;
+using Uri = Android.Net.Uri;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Screenshot
+    {
+        static bool PlatformCanCapture => !Platform.WindowManager.DefaultDisplay.Flags.HasFlag(DisplayFlags.Secure);
+
+        static string GetTempFileName() => Path.Combine(Environment.ExternalStorageDirectory.Path, Path.GetTempFileName());
+
+        static async Task<MediaFile> PlatformCaptureAsync()
+        {
+            var path = GetTempFileName();
+            var view = Platform.GetCurrentActivity(false).Window.DecorView.RootView;
+            view.DrawingCacheEnabled = true;
+            var bitmap = Bitmap.CreateBitmap(view.GetDrawingCache(true));
```

Comment:
The build is failing because  Screenshot\Screenshot.android.cs(31,46): error CS0618: 'View.GetDrawingCache(bool)' is obsolete: 'deprecated' (locally and in CI)